scheduler/src/cook/hooks/launch.clj

```diff
@@ -0,0 +1,150 @@
+;;
+;; Copyright (c) Two Sigma Open Source, LLC
+;;
+;; Licensed under the Apache License, Version 2.0 (the "License");
+;; you may not use this file except in compliance with the License.
+;; You may obtain a copy of the License at
+;;
+;;  http://www.apache.org/licenses/LICENSE-2.0
+;;
+;; Unless required by applicable law or agreed to in writing, software
+;; distributed under the License is distributed on an "AS IS" BASIS,
+;; WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+;; See the License for the specific language governing permissions and
+;; limitations under the License.
+;;
+(ns cook.hooks.launch
+  (:require [clj-time.core :as t]
+            [clj-time.periodic]
+            [chime :as chime]
+            [cook.cache :as ccache]
+            [cook.config :refer [config]]
+            [cook.datomic :as datomic]
+            [cook.hooks.definitions :refer [JobLaunchFilter check-job-launch]]
+            [cook.hooks.util]
+            [mount.core :as mount]
+            [clojure.tools.logging :as log])
+  (:import (com.google.common.cache Cache CacheBuilder)
+           (java.util.concurrent TimeUnit)))
+
+(def default-accept
+  "A default accept object with an expiration at infinity"
+  {:status :accepted :message "No hook object defined" :cache-expires-at (t/date-time 2783 12 03)})
```

Comment:
What is this date?

Reply:
positive infinity. But I wanted it to be something greppable.

Reply:
Hmm, to me the question about why it's so specific is not worth the `grep`-ability